text/0000-roadmap-2019.md

```diff
@@ -0,0 +1,328 @@
+borrow checker), and a new library focusing on name resolution. Eventually,
+we will merge this new front-end with the remaining bits of back-end from
+rustc.
+
+[Chalk]: https://github.com/rust-lang-nursery/chalk
+[Polonius]: http://github.com/rust-lang-nursery/polonius
+[rust-analyzer project]: https://github.com/rust-analyzer/rust-analyzer
+
+## Crates.io
+
+One of the newer teams, the Crates.io Team is going to be focused on growing
+itself, paying down technical debt in the codebase, and establishing themes
+for long-term priorities.
+
+## Dev Tools
+
+The Dev Tools team has a few core functions:
+
+- liaising with the wider project
+- being a bit of a "commons" space for cross-cutting tools concerns
+- finding out what needs are yet to be addressed by tooling and filling them
+- providing some kind of quorum helping smaller tool subteams make decisions
+
+They'll be working to fulfill these functions this year, along with looking
+for ways to improve their internal processes.
+
+Individual devtools subteams may be coming up with their own roadmaps, you
+can see the one of Cargo above, and the IDE team
+[here](https://github.com/rust-dev-tools/dev-tools-team/blob/master/roadmaps/ides-and-editors-2019.md).
+
+## Documentation
+
+The documentation team is going to be completely reformulating itself, and
+possibly re-naming itself as the "learning team." They're working on an RFC
+to do so, and that will lay out the roadmap. Stay tuned!
+
+## Language
+
+The language team has four areas of interest this year:
+
+- **Organizational:**
+    - [Introduce working groups](http://smallcultfollowing.com/babysteps/blog/2019/02/22/rust-lang-team-working-groups/) with the goal of increasing transparency and lowering the barrier to get involved 
+    - Create a [lang-team repository](https://github.com/rust-lang/lang-team/) to help track what is happening and how to get involved
+- **Key ergonomic improvements:**
+    - Async/await in particular is needed to help "unlock" Async I/O
+- **Finish long-standing features:**
+    - Rust has a number of in-progress features -- such as const generics, Generic Associated Types, and specialization -- that have been in development for some time. It's time to finish their designs and ship them.
+- **Reference material and guidelines:**
+    - Work with the documentation team on the Rust reference.
+    - Produce "unsafe code guidelines" that describe what unsafe code can and cannot do.
+
+## Library
+
+The library team will be focusing on maintaining the standard library. The
+team will handle specialized tasks such as overseeing and reviewing ports of
+the standard library to new platforms, as well as typical bug fixes and
+performance improvements.
+
+The team doesn't currently have the bandwidth to take on a large number of
+new features for the standard library, but is specifically planning to
+develop a plan of attack for custom allocators associated with instances of
+collections (e.g. `Vec<T, A: Alloc>`).
```

Comment:
cc @rust-lang/release Do we want to say anything here?

(also @rust-lang/infra is missing but y'all presumably have plans re. CI?)

Reply:
Both teams opted to not add anything specific.

Reply:
@steveklabnik Well... at least for T-Release iirc we didn't really discuss the RFC itself... [more like the roadmap in general](https://paper.dropbox.com/doc/Release-Team-Roadmap--AY7UDwz3BXPr6nChsU1pxWenAg-ScEVi68FA0KmizLQpFe75); but we probably should do that next Wednesday. :)

Reply:
I checked with @Mark-Simulacrum and @aidanhs and they both felt like they didn't want to add anything, but I'm certainly game to include plans that exist. =)